ethereum/contracts/pyth/PythInternalStructs.sol

```diff
@@ -9,38 +9,9 @@ import "@pythnetwork/pyth-sdk-solidity/PythStructs.sol";
 contract PythInternalStructs {
     using BytesLib for bytes;
 
-    struct BatchPriceAttestation {
```

Comment:
it's safe to delete these structs because they're never stored anywhere or shared with users outside the contract?

Reply:
yes